### ISSUE TITLE
Hight code injection

### ISSUE THREAD

**Issue by langelhc** [nodeca/js-yaml#506]
When I run `$ npm audit` I get:

```
  High            Code Injection

  Package         js-yaml

  Patched in      >=3.13.1

  Dependency of   gulp-cssnano

  Path            gulp-cssnano > cssnano > postcss-svgo > svgo > js-yaml

  More info       https://nodesecurity.io/advisories/813


  Moderate        Denial of Service

  Package         js-yaml

  Patched in      >=3.13.0

  Dependency of   gulp-cssnano

  Path            gulp-cssnano > cssnano > postcss-svgo > svgo > js-yaml

  More info       https://nodesecurity.io/advisories/788```

**Comment by puzrin** [comment]
I'm not sure here is a right place to post about your not updated dependencies.

**Comment by cyrus-za** [comment]
@puzrin 3.13.1 also gives that audit error, so its not patched in 3.13.1 which is the latest current package

**Comment by puzrin** [comment]
I think you could check commits history prior to post such statements to this tracker.
